changes.md

```diff
@@ -1,10 +1,14 @@
 # API Changes
 
+## Passes
+* `LikeReplacement` pass must be run by all transformers.
```

Comment:
I think it'd be better to have reference implementations for `BroadcastLike` and `ScalarConstantLike`, and that other backends run the elimination pass if they want to. `INTERPRETER` should run on a structurally unmodified graph.

Reply:
These are not really ops. For example, the op ordering for the Like argument is only needed for shape inference, not execution order, which is why it goes away afterwards. They could be derived from Node directly once we're no longer calling things in the constructors.